lighthouse-core/gather/driver.js

```diff
@@ -522,6 +531,37 @@ class Driver {
     });
   }
 
+  /**
+   * Returns a promise that resolve when a frame has a FCP.
+   */
+  _waitForFCP() {
```

Comment:
I'm personally a fan of the explicit return types both for safety (tsc will happily start unioning things in cases of mismatched early returns, etc) and for documentation purposes (intellisense hover shouldn't be required :).

We should probably get into that at some point because I've definitely noticed you leaving them off more recently :)

Reply:
Yeah I've been leaving it off wherever I see no ambiguous branching going on :)

This one isn't bad though I'll add it.